src/local.c

```diff
@@ -140,87 +140,102 @@ int
 update_localdb(int verbose)
 {
     struct stat s;
-    char tmp[STRBUFSIZ];
-    char outpath[STRBUFSIZ];
-    char outfile[STRBUFSIZ];
-    char outhome[STRBUFSIZ];
+    char tldr_home[STRBUFSIZ];         /* $HOME/TLDR_HOME                      */
+    char tldr_home_db[STRBUFSIZ];      /* $HOME/TLDR_HOME/TLDR_DIR             */
+    char temp_dir[STRBUFSIZ];          /* $HOME/TLDR_HOME/TMP_DIR              */
+    char update_zip_file[STRBUFSIZ];   /* $HOME/TLDR_HOME/TMP_DIR/TMP_FILE     */
+    char update_contents[STRBUFSIZ];   /* $HOME/TLDR_HOME/TMP_DIR/TLDR_ZIP_DIR */
```

Comment:
```suggestion
    char extracted_contents[STRBUFSIZ];   /* $HOME/TLDR_HOME/TMP_DIR/TLDR_ZIP_DIR */
```

Think we could rename this variable to `extracted_contents` or `zip_contents`?